### PR TITLE
Tooltip component

### PR DESCRIPTION
**Pull request by dafuga**

### CODE REVIEW

File: src/lib/components/tooltip/tooltip.svelte [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	} = createTooltip({
+		openDelay,
+		closeDelay
+	});
+</script>
+
+<span use:melt={$trigger}>
+	{#if children}
+		{@render children()}
+	{/if}
```
Comment (deansallinen) [inline src/lib/components/tooltip/tooltip.svelte:37]: Should we default to an info icon if no children?

Comment (deansallinen) [reply src/lib/components/tooltip/tooltip.svelte:37]: Then you can just do `<Tooltip content="asdf" />`

File: src/routes/[network]/(dev)/debug/components/sections/tooltips.svelte [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	import Tooltip from '$lib/components/tooltip/tooltip.svelte';
+</script>
+
+<Stack id="tooltips" class="space-y-8">
+	<div>
+		<h2 class="h2 mb-4">Tooltips</h2>
+		<p>Tooltips provide additional information when hovering over an element.</p>
+	</div>
+
+	<div class="flex items-center space-x-4">
```
Comment (deansallinen) [inline src/routes/[network]/(dev)/debug/components/sections/tooltips.svelte:13]: nit: not sure these surrounding divs are contributing anything

File: src/routes/[network]/(dev)/debug/components/sections/tooltips.svelte [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		<Tooltip
+			content="This is a long tooltip with an icon. Icons can be used to provide visual cues about the type of information being presented. For example, this icon might indicate that the tooltip contains important or critical information that the user should pay attention to."
+			icon={Info}
+		>
+			<span class="cursor-help underline">Tooltip with icon</span>
+		</Tooltip>
+	</div>
```
Comment (deansallinen) [inline src/routes/[network]/(dev)/debug/components/sections/tooltips.svelte:42]: Maybe we add an example with an info icon as the trigger?

**Review by deansallinen — APPROVED** [review]
Looks good!

File: src/lib/components/tooltip/tooltip.svelte (hunk 1 above)
Comment (dafuga) [reply src/lib/components/tooltip/tooltip.svelte:37]: Good idea 👍

File: src/routes/[network]/(dev)/debug/components/sections/tooltips.svelte (hunk 2 above)
Comment (dafuga) [reply src/routes/[network]/(dev)/debug/components/sections/tooltips.svelte:13]: you're absolutely right 😂 I no longer need those classes.

**Pull request merged**